src/engine/test/assets/schemas/custom-ecs-field.json

```
@@ -4774,6 +4774,267 @@
+      "ecs_type": "keyword",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.internal_query_pos": {
+      "description": "Character count of the internal query (if any).\n",
+      "ecs_type": "long",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.context": {
+      "description": "Error context.\n",
+      "ecs_type": "keyword",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.query_pos": {
+      "description": "Character count of the error position (if any).\n",
+      "ecs_type": "long",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.location": {
+      "description": "Location of the error in the PostgreSQL source code (if log_error_verbosity is set to verbose).\n",
+      "ecs_type": "keyword",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.application_name": {
+      "description": "Name of the application of this event. It is defined by the client.\n",
+      "ecs_type": "keyword",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.backend_type": {
+      "description": "Type of backend of this event. Possible types are autovacuum launcher, autovacuum worker, logical replication launcher, logical replication worker, parallel worker, background writer, client backend, checkpointer, startup, walreceiver, walsender and walwriter. In addition, background workers registered by extensions may have additional types.\n",
+      "ecs_type": "keyword",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.error.code": {
+      "description": "Error code returned by Postgres (if any). Deprecated: errors can have letters. Use sql_state_code instead.\n",
+      "ecs_type": "alias",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.timezone": {
+      "description": "Not available",
+      "ecs_type": "alias",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.user": {
+      "description": "Not available",
+      "ecs_type": "alias",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.level": {
+      "description": "Valid values are DEBUG5, DEBUG4, DEBUG3, DEBUG2, DEBUG1, INFO, NOTICE, WARNING, ERROR, LOG, FATAL, and PANIC.\n",
+      "ecs_type": "alias",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    },
+    "postgresql.log.message": {
+      "description": "Not available",
+      "ecs_type": "alias",
+      "type": [
+        "string",
+        "number",
+        "boolean"
+      ]
+    }
```

Invalid JSON, array element should be separated with a '`,`'

```suggestion
    },
```

---

yup